tests/integration/testdata/buildcmd/terraform/image_based_lambda_functions_local_backend/main.tf

```diff
@@ -114,16 +114,17 @@ resource "null_resource" "sam_metadata_aws_lambda_function_l2_function" {
 
 ## Serverless TF
 module "docker_image" {
-  source = "git::https://github.com/moelasmar/terraform-aws-lambda.git//modules/docker-build?ref=master_sam_cli_integration_null_resource_solution"
-
+  source  = "terraform-aws-modules/lambda/aws//modules/docker-build"
```

Comment:
Is there anything preventing this merge?

Reply:
no .. Serverless module is already installed to Terraform registry